pom.xml

```diff
@@ -41,4 +41,25 @@
       <url>http://repo.jenkins-ci.org/public/</url>
     </pluginRepository>
   </pluginRepositories>
+  <dependencies>
+      <dependency>
+      <groupId>org.jenkins-ci.plugins</groupId>
+      <artifactId>beer</artifactId>
```

Comment:
Yes, changing it.

El 10/8/2016 19:55, "Ryan Campbell" <notifications@github.com> escribió:

> In pom.xml
> <https://github.com/jenkinsci/variant-plugin/pull/1#discussion_r74295737>:
>
> > @@ -41,4 +41,25 @@
> >        <url>http://repo.jenkins-ci.org/public/</url>
> >      </pluginRepository>
> >    </pluginRepositories>
> > +  <dependencies>
> > +      <dependency>
> > +      <groupId>org.jenkins-ci.plugins</groupId>
> > +      <artifactId>beer</artifactId>
>
> Shouldn't this be limited to the test scope?
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/jenkinsci/variant-plugin/pull/1/files/f981acf5f178eaa50a495e57ef064584d8d91f1a#r74295737>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/AQXNBN7uKBuyUD4UiOxph0mbTiSRbDZDks5qehB0gaJpZM4JhYzN>
> .
>